float64[] point
---
bool is_point_clear
